draft-ietf-httpbis-cache-latest.xml

```diff
@@ -577,9 +577,9 @@
    When a cache receives a request that can be satisfied by a stored response
    that has a <x:ref>Vary</x:ref> header field (<xref target="field.vary"/>),
    it &MUST-NOT; use that response unless all the selecting header fields
-   nominated by the Vary header field match in both the original request
+   nominated by the Vary header field in both the original request
    (i.e., that associated with the stored response), and the presented
-   request.
+   request, match.
```

Comment:
I don't think this is an improvement.

Reply:
I was confused about "match in both", since "match in both X and Y" would imply that there is some other third thing that both are matched against.  I'll try again...